qcut/docs/task/organize-project-integration.md

```
@@ -0,0 +1,504 @@
+
+**Related Files:**
+- `apps/web/src/lib/__tests__/media-organizer.test.ts` (NEW)
+- `apps/web/src/lib/__tests__/media-usage-analyzer.test.ts` (NEW)
+- `apps/web/src/stores/__tests__/folder-persistence.test.ts` (NEW)
+
+---
+
+## Implementation Order
+
+| Order | Subtask | Priority | Est. Time | Dependencies |
+|-------|---------|----------|-----------|--------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)